sources/updater/waves/accelerated-waves.toml

```diff
@@ -0,0 +1,24 @@
+# NOTE: Update waves have an implicit start time. The initial wave starts
```

Comment:
I don't see the benefit of repeating this in every file.  The README should help general understanding, and the description of the wave structure just below this should clarify intent of this file.